docs/assets/sass/nhsuk.scss

```diff
@@ -0,0 +1,643 @@
+// ==========================================================================
+// New 'beta' header
+// exposing search on mobile
+// ==========================================================================
+@media (max-width: 40.0525em) {
+  .beta-header__search-form {
+    padding: 0 16px 17px;
+    position: relative;
+  }
+  .beta-header__search-wrap::after {
+    background: #005eb8;
+    content: ' ';
+    height: 57px;
+    left: -16px;
+    position: absolute;
+    width: calc(100% + 32px);
+  }
+  .beta-search__submit {
+    background-color: #f0f4f5;
+    border: 0;
+    height: 40px;
+    z-index: 1;
+  }
+  .beta-search__submit .beta-icon__search {
+    fill: #005eb8;
+    height: 27px;
+    width: 27px;
+  }
+  .beta-search__submit:hover {
+    background-color: #003d78;
+    border: 1px solid #ffffff;
+  }
+  .beta-search__submit:hover .nhsuk-icon {
+    fill: #ffffff;
+  }
+  .nhsuk-header__menu-toggle {
+    right: 0;
+  }
+  .nhsuk-search__input {
+    border: 1px solid #ffffff;
+    height: 40px;
+  }
+}
+
+@media (min-width: 48.0625em) {
+  .nhsuk-header__navigation {
```

Comment:
Example of a `nhsuk-` prefix that might interfere with prototypes using the Header component, but only should be being used on the NHS website Header